hammer/library/aws/security_groups.py

```diff
@@ -8,12 +8,14 @@
 from library.utility import jsonDumps
 from library.aws.s3 import S3Operations
 from library.aws.utility import convert_tags
+from library.config import Config
 
 
 class RestrictionStatus(Enum):
     Restricted = "restricted"
     OpenCompletely = "open_completely"
     OpenPartly = "open_partly"
+    SafeIP = "safe_ips"
```

Comment:
Maybe just `safe` to be consistent in naming?

Reply:
Updated and tested.